<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_SEOverview</class>
 <widget class="QDialog" name="Dialog_SEOverview">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SE Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>451</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ImportSEProblemSheet">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>180</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>导入问题单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SelectSEProblemSheet">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>140</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>选择问题单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ExportOverviewSheet">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>220</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>导出汇总表</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>581</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>汇总表类型</string>
   </property>
   <widget class="QRadioButton" name="radioButton_Project">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Project</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Engineer">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Engineer</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>481</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>SE问题单清单</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>591</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>进度</string>
   </property>
   <widget class="QProgressBar" name="progressBar_SE">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>591</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>300</y>
     <width>75</width>
     <height>52</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_SEOverview</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_SEOverview</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
